migration/mysql/20240624041010_initialization.sql

```diff
@@ -0,0 +1,54 @@
+-- Create "organization" table
+CREATE TABLE `organization` (`id` varchar(255) NOT NULL, `name` varchar(255) NOT NULL, `url_code` varchar(255) NOT NULL, `description` text NOT NULL, `disabled` bool NOT NULL DEFAULT 0, `archived` bool NOT NULL DEFAULT 0, `trial` bool NOT NULL DEFAULT 0, `system_admin` bool NOT NULL DEFAULT 0, `created_at` bigint NOT NULL, `updated_at` bigint NOT NULL, PRIMARY KEY (`id`), UNIQUE INDEX `unique_url_code` (`url_code`)) CHARSET utf8mb4 COLLATE utf8mb4_bin;
+-- Create "webhook" table
+CREATE TABLE `webhook` (`id` varchar(255) NOT NULL, `name` varchar(255) NOT NULL, `description` text NOT NULL, `created_at` bigint NOT NULL, `updated_at` bigint NOT NULL, `environment_namespace` varchar(255) NOT NULL, PRIMARY KEY (`id`), INDEX `idx_environment_namespace` (`environment_namespace`)) CHARSET utf8mb4 COLLATE utf8mb4_bin;
+-- Create "feature_last_used_info" table
+CREATE TABLE `feature_last_used_info` (`id` varchar(255) NOT NULL, `feature_id` varchar(255) NOT NULL, `version` bigint NOT NULL, `last_used_at` bigint NOT NULL, `client_oldest_version` varchar(255) NOT NULL, `client_latest_version` varchar(255) NOT NULL, `created_at` bigint NOT NULL, `environment_namespace` varchar(255) NOT NULL, PRIMARY KEY (`id`, `environment_namespace`)) CHARSET utf8mb4 COLLATE utf8mb4_bin;
+-- Create "admin_audit_log" table
+CREATE TABLE `admin_audit_log` (`id` varchar(255) NOT NULL, `timestamp` bigint NOT NULL, `entity_type` int NOT NULL, `entity_id` varchar(255) NOT NULL, `type` int NOT NULL, `event` json NOT NULL, `editor` json NOT NULL, `options` json NOT NULL, PRIMARY KEY (`id`), INDEX `idx_timestamp_desc` (`timestamp` DESC)) CHARSET utf8mb4 COLLATE utf8mb4_bin;
+-- Create "admin_subscription" table
+CREATE TABLE `admin_subscription` (`id` varchar(255) NOT NULL, `created_at` bigint NOT NULL, `updated_at` bigint NOT NULL, `disabled` bool NOT NULL DEFAULT 0, `source_types` json NOT NULL, `recipient` json NOT NULL, `name` varchar(255) NOT NULL, PRIMARY KEY (`id`)) CHARSET utf8mb4 COLLATE utf8mb4_bin;
+-- Create "api_key" table
+CREATE TABLE `api_key` (`id` varchar(255) NOT NULL, `name` varchar(255) NOT NULL, `role` int NOT NULL, `disabled` bool NOT NULL DEFAULT 0, `created_at` bigint NOT NULL, `updated_at` bigint NOT NULL, `environment_namespace` varchar(255) NOT NULL, PRIMARY KEY (`id`, `environment_namespace`)) CHARSET utf8mb4 COLLATE utf8mb4_bin;
+-- Create "atlas_schema_revisions" table
+CREATE TABLE `atlas_schema_revisions` (`version` varchar(255) NOT NULL, `description` varchar(255) NOT NULL, `type` bigint unsigned NOT NULL DEFAULT 2, `applied` bigint NOT NULL DEFAULT 0, `total` bigint NOT NULL DEFAULT 0, `executed_at` timestamp NOT NULL, `execution_time` bigint NOT NULL, `error` longtext NULL, `error_stmt` longtext NULL, `hash` varchar(255) NOT NULL, `partial_hashes` json NULL, `operator_version` varchar(255) NOT NULL, PRIMARY KEY (`version`)) CHARSET utf8mb4 COLLATE utf8mb4_bin;
```

Comment:
`atlas_schema_revisions` table isn't need to be migrated.
It will generated by atlas automatically.